Makefile

```diff
@@ -233,10 +233,15 @@ undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/confi
 	make deploy-yaml | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
 
 .PHONY: deploy-example
-deploy-example: ## Deploy an example Istio resource
+deploy-example: ## Deploy an example Istio resource on OpenShift
```

Comment:
perhaps we could rename this target to be `deploy-example-openshift` to keep the pattern?

Reply:
done